services/keywords/utils.py

```diff
@@ -0,0 +1,60 @@
+from google.cloud import language # pylint: disable=import-error
+from google.cloud.language import enums, types # pylint: disable=import-error
+from google.api_core import exceptions
+import constants
+
+def gcloud_syntax_extraction(text):
+    """
+    Calls the gcloud natural langauge api's analyze syntax
+    Attributes:
+        text: the text which we wish to analyze
+    Returns:
+        the response given by the api call
+    """
+    try:
+        client = language.LanguageServiceClient()
+        # pylint: disable=no-member
+        document = types.Document(
+            content=text,
+            type=enums.Document.Type.PLAIN_TEXT)
+        response = client.analyze_syntax(document=document)
+    except exceptions.GoogleAPICallError as err:
+        return {"error": {"type": "GoogleAPICallError", 
```

Comment:
So this `gcloud_syntax_extraction` shouldn't need to know about your API's agreed-upon `error` return. I'd recommend deferring that to the calling func `extract_keywords` and just re-raising the error here (or not catching it at all). That will also save you a lot of copy-pasting.